python/metatensor-operations/tests/sort.py

```diff
@@ -124,31 +126,87 @@ def tensor_sorted():
     return TensorMap(keys, [block_1, block_2])
 
 
-def test_sort(tensor, tensor_sorted):
+@pytest.fixture
+def tensor_sorted_descending():
+    block_1 = TensorBlock(
+        values=np.array([[3, 5], [1, 2]]),
+        samples=Labels(["s"], np.array([[2], [0]])),
+        components=[],
+        properties=Labels(["p"], np.array([[1], [0]])),
+    )
+
+    # fmt: off
```

Comment:
this should not be needed

Reply:
not needed but black makes this really hard to read so I turned it off

Reply:
This have strong feelings on. This sort test is already hard to understand as it is, and the black formatting made it much harder.

Reply:
Unlike other tests, the order must be much easier to read so one can follow what is how moved. I added comments why it is turned off.